src/blog.njk

```diff
@@ -2,13 +2,12 @@
 layout: nohero
 sitemapPriority: 0.9
 pagination:
-  data: collections.posts
+  data: collections.livePosts
```

Comment:
You'll need to do this for all the categories:

![Screenshot 2024-01-05 at 12 53 24](https://github.com/FlowFuse/website/assets/2529595/0a1b69a3-9bed-47a5-b155-accc207bc438)